atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolImpl.java

```diff
@@ -132,32 +129,32 @@ static CassandraClientPoolImpl createImplForTest(
         return cassandraClientPool;
     }
 
-    public static CassandraClientPool create(MetricsManager metricsManager, CassandraKeyValueServiceConfig config) {
-        return create(metricsManager,
+    @VisibleForTesting
+    static CassandraClientPoolImpl createImplForTest(
```

Comment:
more test factories 😢 😢

Reply:
I know, it's horrible :( Maybe we should add this to the list of things that could do with a refactoring?